cmd/omctl/cmd/root.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
+	Use:          "omctl",
+	Short:        "CLI for openmock",
+	SilenceUsage: true,
+	Long: `
+		A simple CLI for interacting with openmock.
+`,
```

Comment:
Wonder if we should add a [PersistentPreRun](https://github.com/spf13/cobra#prerun-and-postrun-hooks) or something to check if we actually can connect to `openMockUrl` before loading the templates? Right now I see that templates are loaded even if the POST is eventually going to fail. 

Not a big deal for now though, if it fails it fails.

Reply:
I've added a `PersistentPreRunE` that gets the remote templates first to see if it's reachable.